src/jsMain/kotlin/app/softwork/bootstrapcompose/Radio.kt

```diff
@@ -0,0 +1,59 @@
+package app.softwork.bootstrapcompose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import kotlinx.uuid.UUID
+import org.jetbrains.compose.web.attributes.*
+import org.jetbrains.compose.web.dom.Div
+import org.jetbrains.compose.web.dom.Input
+import org.jetbrains.compose.web.dom.Label
+import org.jetbrains.compose.web.dom.Text
+
+@Composable
+public fun RadioGroup(content: @Composable RadioGroupScope.() -> Unit) {
```

Comment:
Does it needs to be `@Composable`? In `Input` it is not.

Reply:
Yes, it won't compile without it. I believe the reason is because within this function calls are made to RadioGroupScope.Radio(), which is Composable because it directly creates Divs.